python_lib/setup_mlir.sh

```diff
@@ -0,0 +1,15 @@
+alias sed=/opt/homebrew/opt/gsed/bin/gsed
```

Comment:
This script was a testing setup script during initial development. If it is no longer used should it be removed?

Reply:
Removed in https://github.com/openqasm/qe-compiler/pull/264/commits/8f588c934e1bd4b5af463a2a568b384d6d764690